core/src/main/java/bisq/core/dao/DaoFacade.java

```diff
@@ -680,4 +685,27 @@ public double getRequiredThreshold(Proposal proposal) {
     public Coin getRequiredQuorum(Proposal proposal) {
         return proposalService.getRequiredQuorum(proposal);
     }
+
+    public long getRequiredBond(Optional<RoleProposal> roleProposal) {
```

Comment:
I feel these functions should reside in DaoStateService for consistency. This works, most other actual work is done where it's more relevant rather than in DaoFacade itself.

Reply:
Hm, not sure. Proposal is another domain as DaoState so I prefer to keep it contained in ProposalService. Sure there is much overlap but DaoStateService is already huge so in doubt I prefer to keep it in ProposalService. With DaoFacade I am not sure if we should keep it mid term. Its quite boilerplate.

Reply:
Just leave it for now. But would be nice to find a better place for it eventually.